### ISSUE TITLE
why there is no training loss or validation loss with the script of fine-tune?

### ISSUE THREAD

**Issue by abdelkareemkobo** [tonywu71/colpali-cookbooks#18]
I want to fine-tune the colqwen, i followed your notebook about fine-tune colpali and replaced the model, processor with colqwen2.5 but i didn't see any training loss or validation losss with eval_results and with  train_results:

Step	Training Loss	Validation Loss	Model Preparation Time
1	No log	0.000000	0.015700
2	No log	0.000000	0.015700
3	No log	0.000000	0.015700
